velox/exec/fuzzer/PrestoQueryRunner.cpp

```diff
@@ -212,13 +212,58 @@ std::string toTypeSql(const TypePtr& type) {
   }
 }
 
-std::string toCallSql(
+std::string toCallSql(const core::CallTypedExprPtr& call) {
+  std::stringstream sql;
+  sql << call->name() << "(";
+  for (auto i = 0; i < call->inputs().size(); ++i) {
```

Comment:
Is this loop copy-pasted in 2 places? If so, let's refactor to avoid that.

Reply:
Fixed by adding a `toCallSqlInternal` helper method.